portfolio/src/main/java/com/google/sps/servlets/NewCommentServlet.java

```diff
@@ -25,18 +25,24 @@
 import com.google.appengine.api.datastore.DatastoreService;
 import com.google.appengine.api.datastore.DatastoreServiceFactory;
 import com.google.appengine.api.datastore.Entity;
+import java.time.format.DateTimeFormatter;
+import java.time.*; 
 
-/** Servlet that returns some example content. TODO: modify this file to handle comments data */
+/* Servlet that creates a comment*/
 @WebServlet("/new-comment")
 public class NewCommentServlet extends HttpServlet {
   
+  private static final DateTimeFormatter commentDateFormat = DateTimeFormatter.ofPattern("yyyy/MM/dd");
+  
   @Override
   public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
     String text = request.getParameter("commentText");
     long timestamp_millis = System.currentTimeMillis();
-
+    String currentDate = commentDateFormat.format(LocalDateTime.now());
+    
     Entity commentEntity = new Entity("Comment");
     commentEntity.setProperty("content", text);
+    commentEntity.setProperty("currentDate", currentDate);
     commentEntity.setProperty("timestamp_millis", timestamp_millis);
```

Comment:
I don't think you need the individual variables above. You can do the following:

```
    commentEntity.setProperty("content", request.getParameter("commentText"));
    commentEntity.setProperty("currentDate", commentDateFormat.format(LocalDateTime.now()));
    commentEntity.setProperty("timestamp_millis", timestamp_millis);
```